pkg/metrics/metrics.go

```diff
@@ -107,21 +107,34 @@ The label 'result' can have the following values:
 		}, []string{"cluster_queue"},
 	)
 
+	EvictedWorkloads = prometheus.NewGaugeVec(
```

Comment:
Please add some test for the metric, either unit test or integration. I know we have some integration tests for metrics already, so extending an existing integration test to assert of the metrics sounds good to me.

Reply:
Thanks. My last ask would be that the unit test verifies `ReportEvictedWorkloads` works ok, still it would be good to assert in integration tests to make sure it is called.